sklearn/utils/estimator_checks.py

```diff
@@ -131,6 +131,29 @@ def _yield_classifier_checks(name, Classifier):
     if 'class_weight' in Classifier().get_params().keys():
         yield check_class_weight_classifiers
 
+def check_supervised_y_no_nan(name, Estimator):
+    # Checks that the Estimator targets are not NaN.
+
+    np.random.seed(888)
+    X = np.random.randn(10, 5)
```

Comment:
@MechCoder: I thought that `np.random.seed(888)` would provide a sufficient random number generator; I didn't know that it would seed the global, so sorry about that!

I fixed this in 290e0ea.